### DISCUSSION TITLE
Teamlead sees Timesheet from Project where he is only Teammember

### DISCUSSION THREAD

**Discussion by jens-2305** [kimai/kimai#3703] [Questions & Answers] [answered]
Hi, I have an example at
 
https://demo-empty.kimai.org/
 
to explain my problem.

I have 2 projects with the following team members

gbs Project 1 (gbs.team1[Teamlead], gbs.m1[Teammember], gbs.m2[Teammember], gbs.team2[Teammember])

gbs Project 2 (gbs.team2[Teamlead], gbs.m1[Teammember])

If the gbs.team2 now looks at the time tracking (Administration->Time tracking) he unfortunately also sees the time tracking of gbs.m1 from the gbs Project 1. But this is not correct because he is actually only allowed to see the time tracking of the project 2 where he is teamlead or ? Password for all mentioned users is gbsstart01

**Comment by kevinpapst** [comment, marked as answer]
No. Kimai does not distinguish that. This works as expected.

**Reply by jens-2305** [reply]
Thank you for your reply. Is there a way to change this easily ? For data protection reasons, the team leader is only allowed to see the projects assigned to him with the times of the employees who are assigned to him.

**Reply by kevinpapst** [reply]
dont allow access to the screen

**Reply by jens-2305** [reply]
but then the teamleader has no way of seeing who has entered what on his project

**Reply by kevinpapst** [reply]
the reports still offer access to that summarized data.

you can also
- create a second user account for that teamlead
- build your own frontend using the api

**Reply by jens-2305** [reply]
yes, unfortunately the summary of the times is not suitable to see possible wrong bookings. i will try to build a report that allows the team leader to see only his projects, maybe i will succeed. thanks for your effort
